lua/entities/gmod_wire_expression2/core/custom/cfc_entity.lua

```diff
@@ -52,4 +52,20 @@ e2function void entity:ejectPodTo(vector pos)
 
     driver:ExitVehicle()
     driver:SetPos(clampedPos)
-end
\ No newline at end of file
+end
+
+e2function number entity:setFairPos(vector target)
+    if this:IsPlayer() then return -3 end -- Debug output -3 if the target is a player
+    local CFCMaxFairSetDist = GetConVar("CFC_Lib_MaxFairSetDistance"):GetFloat()
```

Comment:
maybe just `maxDistance` the cfc part is unnecessary as its a a local variable

Reply:
Agreed